numpy/ma/core.pyi

```diff
@@ -18,13 +18,15 @@ from typing_extensions import TypeIs, TypeVar
 import numpy as np
 from numpy import (
     _AnyShapeT,
+    _ArrayInt_co,
     _HasDType,
     _HasDTypeWithRealAndImag,
     _ModeKind,
     _OrderACF,
     _OrderKACF,
     _PartitionKind,
     _SortKind,
+    _ToIndices,
```

Comment:
This one is a lot beefier, so IMO it's fine to keep this one. (Although it's probably cleaner to put this and others like it in `numpy._typing`, but that's not very high on the priority list, and of course out-of-scope)